presto-mysql/src/test/java/io/prestosql/plugin/mysql/DockerizedMySqlServer.java

```diff
@@ -40,16 +39,15 @@
 
     public DockerizedMySqlServer()
     {
-        this.dockerContainer = new DockerContainer(
-                "mysql:8.0.15",
-                ImmutableList.of(MYSQL_PORT),
-                ImmutableMap.of(
+        this.dockerContainer = DockerContainer.builder("mysql:8.0.15")
```

Comment:
It would look better if `image` was provider via regular setter. (I'm aware it's a required param.)

Reply:
Maybe we could rename this method?
```
this.dockerContainer = DockerContainer.image("mysql:8.0.15")
        .setPorts(MYSQL_PORT)
        .setHealthCheck(DockerizedMySqlServer::healthCheck)
        .build();

Reply:
Might do. `forImage`?

Reply:
Thanks, it looks much better now.